documentation/modules/configuring/con-config-mirrormaker2-connectors.adoc

```diff
@@ -40,7 +40,7 @@ The following table describes connector properties and the connectors you config
 |✓
 |
 
-|offset-syncs.topic.location:: The location of the `offset-syncs` topic, which can be the `source` (default) or `target` cluster. If you set the property to `target`, read-only access to the source cluster is granted to MirrorMaker.
+|offset-syncs.topic.location:: The location of the `offset-syncs` topic, which can be the `source` (default) or `target` cluster.
```

Comment:
Is the setting `source` and `target`? Or are these the aliases of the source and target clusters?

Reply:
Yes this config is set to either `source` or `target`. It is not using the cluster aliases.